javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -160,6 +166,14 @@ private TreeUtils() {
         } catch (NoSuchMethodException e) {
             throw new AssertionError("Unexpected error in TreeUtils static initializer", e);
         }
+
+        Method jcvardeclDeclaredusingvar;
```

Comment:
I would probably use either a shorter name or follow the capitalization of the method/class.